spec/factories/claim/shared/common_claim_traits.rb

```diff
@@ -0,0 +1,75 @@
+# These methods are going to the global namespace so do not abuse it and
```

Comment:
This comment (I think) related to the `claim_state_common_traits` method, which you have now removed.

Reply:
Although I have named this file as a generic holder of traits common to claims it is, in fact, currently only storing common claim state transition traits/helpers. The comment applies generally to all "global" traits as they can cause clashes, which is why i put the comment in here as well. 

Happy to remove the comment if you think it adds no value?

Reply:
If I understood correctly, the comment was copied over from `spec/factories/claim/base_claims.rb` where it was referring to the `claim_state_common_traits` methods (which contained these trait definitions) as well as a few other methods. It is applicable to the methods, which are still in `base_claims.rb`, but not these traits.

Reply:
anything defined directly inside `FactoryBot.define do` will go into the global namespace and therefore could have suffer collisions i believe. 

I will remove the comment as it is obviously confusing.